assets/common/src/deploy.py

```diff
@@ -260,9 +261,9 @@ def main():
     args = parse_args()
     ml_client = get_mlclient()
     # get registered model id
-    if args.registration_details:
+    if args.model_registration_details:
         model_info = {}
-        with open(args.registration_details) as f:
+        with open(args.model_registration_details) as f:
```

Comment:
It should be **os.path.join(model_registration_details,"model_registration_details")** no??